source/Mission.cpp

```diff
@@ -1107,6 +1130,7 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 	
 	// Generate the substitutions map.
 	map<string, string> subs;
+	subs["<uuid>"] = result.uuid;
```

Comment:
Do we ever want to display the UUID in-game?
I would expect the UUID not to show up on the UI anywhere.

Reply:
Removed.

It was for debugging. I suppose if I need to debug again, I can put that line back in manually.